plugins/inventory/nb_inventory.py

```diff
@@ -943,6 +944,12 @@ def extract_cluster(self, host):
         except Exception:
             return
 
+    def extract_cluster_device(self, host):
+        # cluster device does not have a slug
+        if host.get("device", None) is None:
+            return ""
+        return host.get("device", {"name": ""}).get("name", "")
```

Comment:
This looks awful similar to the above return when there is no device key in the dictionary. 

Overall I think this needs to be improved

Reply:
What is your opinion should we return, if there is no value assigned. Should it be null/None?
Also, is there a handy way to determine the object type of the current host. Then i can only return the cluster_device variable, when the object is an virtual machine.

Reply:
> What is your opinion should we return, if there is no value assigned. Should it be null/None?

I would follow the pattern set by the other functions in this code. Specifically it appears that most would `return` (which is short hand for returning None) although some return an empty list where it makes sense to.